esrally/metrics.py

```diff
@@ -912,7 +912,10 @@ def _add(self, doc):
         self._docs.append(doc)
 
     def _get(self, name, task, operation_type, sample_type, node_name, mapper):
-        query = {"query": self._query_by_name(name, task, operation_type, sample_type, node_name)}
+        query = {
+            "query": self._query_by_name(name, task, operation_type, sample_type, node_name),
+            "size": 10000,  # TODO maybe scroll or something.
```

Comment:
This bit here is required to get back all of the disk usage telemetry. It's not really the right solution though. It needs to be a scroll or multiple queries or something. The default size was tiny and was cutting off many details.

Reply:
It feels rude to error out if your telemetry device fails to build the right output. If the run takes a few hours that'd be devastating. But it feels rude to silently fail - like this does. Is a warning a good idea? I'm not sure I'd see one.